src/requirementslib/fileutils.py

```diff
@@ -92,19 +92,9 @@ def get_long_path(short_path: Text) -> Text:
 
 
 def normalize_path(path):
-    """Return a case-normalized absolute variable-expanded path.
```

Comment:
I'd like us to keep doc strings in place, if you are not adding type annotations.

Reply:
but the implementation changed and I wasn't sure if the comment is still accurate.
Willing to add it back though for me it didn't provide much value.

Reply:
ok.